rules/code-quality/tests/Rector/FuncCall/ArrayKeysAndInArrayToArrayKeyExistsRector/Fixture/fixture.php.inc

```diff
@@ -1,28 +1,48 @@
 <?php
 
-namespace Rector\CodeQuality\Tests\Rector\FuncCall\ArrayKeysAndInArrayToIssetRector\Fixture;
+namespace Rector\CodeQuality\Tests\Rector\FuncCall\ArrayKeysAndInArrayToArrayKeyExistsRector\Fixture;
 
-class SomeClass
+use Rector\CodeQuality\Tests\Rector\FuncCall\ArrayKeysAndInArrayToArrayKeyExistsRector\RunnableInterface;
+
+class SomeClass implements RunnableInterface
 {
-    public function run($packageName, $values)
+    public function test($packageName, $values)
     {
         $keys = array_keys($values);
         return in_array($packageName, $keys, true);
     }
+
+    public function run()
+    {
+        $packageName = "foo";
+        $values      = ["foo" => "bar"];
+
+        return $this->test($packageName, $values);
+    }
```

Comment:
Having 2+ methods in test fixture adds huge complexity to maintenance.

Could there be another way having just one method?

Reply:
Sure, just I can just use `run()` directly.